tests/e2e/tests/devfiles/Scala.spec.ts

```diff
@@ -45,10 +51,10 @@ suite.skip(`${stack} test`, async () => {
     });
 
     suite('Language server validation', async () => {
-        commonLsTests.errorHighlighting(tabTitle, 'Abc:', 21);
-        // commonLsTests.suggestionInvoking(tabTitle, 15, 31, 'Console scala');
-        commonLsTests.autocomplete(tabTitle, 25, 28, 'name: String');
-        // commonLsTests.codeNavigation(tabTitle, 19, 7, codeNavigationClassName, 30_000); // not working
+        commonLanguageServerTests.errorHighlighting(tabTitle, 'Abc:', 21);
+        // commonLanguageServerTests.suggestionInvoking(tabTitle, 15, 31, 'Console scala');
```

Comment:
I know this is only a common rename, just for your attention. Maybe it worth to delete commented lines? It will make code cleaner.

Reply:
Also need to verify the codenavigation and suggestion invoking for Scala and re-enable or delete this. thanks for the notice

Reply:
will also fix as part of https://github.com/eclipse/che/issues/19749